libraries/DHT-sensor-library/DHT-unified-sensor/project.json

```diff
@@ -0,0 +1 @@
+{ "name": "DHT-sensor-library", "author": "Tony DiCola" }
```

Comment:
author should by "Rob Tillaart"  no?

Reply:
![image](https://user-images.githubusercontent.com/68117142/91564340-1f53ba80-e95e-11ea-960d-e2a6c28800d5.png)

Reply:
Oh, it seems like `DHT_Unified_Sensor.ino` here is the wrong one! Can you please look into it?

Reply:
My Bad. Fixed now